bot/src/keeper/surplus.ts

```diff
@@ -0,0 +1,131 @@
+import BigNumber from 'bignumber.js';
+import { SurplusAuctionActive } from 'auctions-core/src/types';
+import { bidToSurplusAuction, enrichSurplusAuction, collectSurplusAuction } from 'auctions-core/src/surplus';
+import getSigner from 'auctions-core/src/signer';
+import { fetchAllowanceAmountMKR, setAllowanceAmountMKR } from 'auctions-core/src/authorizations';
+import { fetchBalanceMKR } from 'auctions-core/src/wallet';
+import { formatToAutomaticDecimalPointsString } from 'auctions-core/src/helpers/formatToAutomaticDecimalPoints';
+import { KEEPER_SURPLUS, KEEPER_SURPLUS_MINIMUM_NET_PROFIT_DAI } from '../variables';
+import { isSetupCompleted } from './setup';
+
+const currentlyExecutedAuctions = new Set();
+
+const checkAndParticipateIfPossible = async function (network: string, auction: SurplusAuctionActive) {
+    // check if setupKeeper hasn't run
+    if (!isSetupCompleted) {
+        return;
+    }
```

Comment:
This is not what it does, it only checks if `isSetupCompleted` (which is function) is not falsy, which is never the case